core/cont/src/TClassTable.cxx

```diff
@@ -630,15 +703,19 @@ TProtoClass *TClassTable::GetProto(const char *cname)
       ::Info("GetDict", "searches for %s", cname);
    }
 
-   if (!CheckClassTableInit()) return nullptr;
+   if (!CheckClassTableInit())
+      return nullptr;
 
    if (gDebug > 9) {
       ::Info("GetDict", "searches for %s", cname);
       fgIdMap->Print();
```

Comment:
Is it ok not to protect `fgIdMap` here?

Reply:
No.  The protection was indeed missing.  Thanks.